src/coreclr/tests/src/Interop/COM/NativeClients/Events/EventTests.cpp

```diff
@@ -16,28 +16,89 @@
 #define THROW_IF_FAILED(exp) { hr = exp; if (FAILED(hr)) { ::printf("FAILURE: 0x%08x = %s\n", hr, #exp); throw hr; } }
 #define THROW_FAIL_IF_FALSE(exp) { if (!(exp)) { ::printf("FALSE: %s\n", #exp); throw E_FAIL; } }
 
-#include <map>
```

Comment:
Is it just the debug iterators causing issues? Can we set `_ITERATOR_DEBUG_LEVEL`?

Reply:
Possibly. However @AndyAyersMS statement above about the state of the CRT being the real issue is why we may want to just remove its usage entirely.